ui/raidboss/data/07-dt/alliance/cloud_of_darkness_chaotic.ts

```diff
@@ -0,0 +1,673 @@
+        const mech = matches.id === '9DFD' ? 'left' : (matches.id === '9DFB' ? 'right' : 'out');
+        const mechOutput = output[mech]!();
+        const followup = data.bladeOfDarknessFollowup;
+        if (followup === undefined)
+          return mechOutput;
+
+        delete data.bladeOfDarknessFollowup;
+
+        return output.combo!({
+          mech: mechOutput,
+          followup: output[followup]!(),
+        });
+      },
+      outputStrings: {
+        combo: {
+          en: '${mech} => ${followup}',
+        },
+        left: {
+          en: 'Left, under hand',
+        },
+        right: {
+          en: 'Right, under hand',
+        },
+        aero: {
+          en: 'Knockback',
+        },
+        death: Outputs.outThenIn,
+        out: Outputs.out,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deluge of Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E3D', '9E01'], source: 'Cloud of Darkness', capture: false },
+      response: Responses.bleedAoe('alert'),
+    },
+    {
+      id: 'Cloud Chaotic Grim Embrace Collector',
+      type: 'StartsUsing',
+      netRegex: { id: ['9E39', '9E3A'], source: 'Cloud of Darkness', capture: true },
+      run: (data, matches) => {
+        data.deadlyEmbraceDodgeDirectionCollected = matches.id === '9E39' ? 'back' : 'front';
+      },
+    },
+    {
+      id: 'Cloud Chaotic Deadly Embrace',
+      type: 'GainsEffect',
+      netRegex: { effectId: '1055', capture: true },
+      condition: Conditions.targetIsYou(),
+      preRun: (data) =>
+        data.deadlyEmbraceDodgeDirection = data.deadlyEmbraceDodgeDirectionCollected,
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 7,
+      durationSeconds: 7,
+      infoText: (data, _matches, output) => output[data.deadlyEmbraceDodgeDirection]!(),
+      outputStrings: {
+        back: {
+          en: 'Bait hand, dodge backwards',
+        },
+        front: {
+          en: 'Bait hand, dodge forwards',
+        },
+        unknown: Outputs.unknown,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Rapid-sequence Particle Beam',
+      type: 'StartsUsing',
+      netRegex: { id: '9E40', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Wild Charge (alliances)',
+        },
+      },
+    },
+    {
+      id: 'Cloud Chaotic Unholy Darkness',
+      type: 'StartsUsing',
+      netRegex: { id: 'A12D', source: 'Cloud of Darkness', capture: false },
+      infoText: (_data, _matches, output) => output.healerGroups!(),
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      id: 'Cloud Chaotic Flare Marker',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.flareMarker, capture: true },
+      condition: Conditions.targetIsYou(),
+      infoText: (_data, _matches, output) => output.flare!(),
```

Comment:
I realize we don't want to overload the player with alerts, but at least from my experience with running week 1, we might want to bump the Flare warning to alert.